### PR TITLE
Scores dynamic mapping

### PR DESCRIPTION
**Pull request by frenchy64**
Epic: https://github.com/advthreat/iroh/issues/7341

We want incident `scores` to be an open map, which can be achieved via dynamic mapping. See explanation in https://github.com/threatgrid/ctim/pull/398 for context.

Will require a migration for INT.

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: Scores dynamic mapping
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

File: src/ctia/entity/incident.clj [hunk 1]
```diff
@@ -126,6 +126,12 @@
 (def incident-mapping
   {"incident"
    {:dynamic false
+    :date_detection false
+    :numeric_detection true
+    :dynamic_templates
+    [{:num
+      {:match_mapping_type "long"
```
Comment (ereteog) [inline src/ctia/entity/incident.clj:133]: Scores could be float.
```suggestion
      {:match_mapping_type "float"
```

Comment (frenchy64) [reply src/ctia/entity/incident.clj:133]: I thought the point of this was to coerce long=>float? IIUC your change is a float=>float coercion?

Comment (ereteog) [reply src/ctia/entity/incident.clj:133]: yes you are totally right, ignore my comment :-)

**Review by ereteog — APPROVED** [review]
FOR THE WIN!

**Pull request merged**